tests/property/ic_api/rejectCode/src/rejector/index.ts

```diff
@@ -15,4 +10,9 @@ export default class {
     rejectWithMessage(): void {
         reject('Rejection with message');
     }
+
+    @query([], undefined, { manual: true })
```

Comment:
Why is this manual and why is it using reply?

Reply:
done